src/gbinder_rpc_protocol.h

```diff
@@ -34,6 +34,7 @@
 #define GBINDER_RPC_PROTOCOL_H
 
 #include "gbinder_types_p.h"
+#include "gbinder_fmq_p.h"
```

Comment:
there's no need to include it from `gbinder_rpc_protocol.h` , `GBinderFmq` is defined in `gbinder_types.h`  and it's enough to compile the definition of the `write_fmq_descriptor` callback.

Reply:
ah, ok. one more reason to make `write_fmq_descriptor` unconditional :)

Reply:
Include removed.